libs/barista-components/table/src/table-module.ts

```diff
@@ -91,13 +98,15 @@ const EXPORTED_DECLARATIONS = [
     CommonModule,
     PortalModule,
     CdkTableModule,
+    DragDropModule,
     DtIconModule,
     DtFormFieldModule,
     DtInputModule,
     DtButtonModule,
     DtIndicatorModule,
     DtFormattersModule,
     DtEmptyStateModule,
+    ReactiveFormsModule,
```

Comment:
I use a form control for validating the input field of the order cell

Reply:
the normal FormsModule should be enough for your use case

Reply:
no, I can't bind to formControl if I switch to the FormsModule, or am I missing something?

Reply:
It's perfectly fine to use the ReactiveFormsModule in my opinion, since we have @angular/forms as a peerDep already - so if you want to use the formControl internally that's ok for me :)